watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/supercritical_sludge_to_gas/supercritical_sludge_to_gas_ui.py

```diff
@@ -0,0 +1,641 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+from watertap.ui.fsapi import FlowsheetInterface
+from watertap.core.util.initialization import assert_degrees_of_freedom
+from watertap.examples.flowsheets.case_studies.wastewater_resource_recovery.supercritical_sludge_to_gas.supercritical_sludge_to_gas import (
+    build,
+    set_operating_conditions,
+    initialize_system,
+    solve,
+    add_costing,
+)
+from pyomo.environ import units as pyunits, assert_optimal_termination
+from pyomo.util.check_units import assert_units_consistent
+
+
+def export_to_ui():
+    return FlowsheetInterface(
+        name="Supercritical sludge to gas",
+        do_export=export_variables,
+        do_build=build_flowsheet,
+        do_solve=solve_flowsheet,
+    )
+
+
+def export_variables(flowsheet=None, exports=None):
+    fs = flowsheet
+    # --- Input data ---
+    # Feed conditions
+    flow_mass = (
+        fs.feed.flow_mass_comp[0, "H2O"]
+        + fs.feed.flow_mass_comp[0, "organic_solid"]
+        + fs.feed.flow_mass_comp[0, "inorganic_solid"]
+        + fs.feed.flow_mass_comp[0, "organic_solid"]
```

Comment:
I am guessing this duplicate was by accident (i.e., you add mass flow for `organic_solid` twice). Is there another component that should be added here or should you just delete the duplicate?

Reply:
Yeah, it should be `organic_liquid`